SYCL/ESIMD/aot_mixed.cpp

```diff
@@ -5,7 +5,8 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: gpu
+// This test passes with 21.49.21786 driver, and fails with older versions.
```

Comment:
The description of PR says the test "fails with newer driver", and this comment says it fails with "older version".

Did the test pass previously and started failing after GPU driver update?

Reply:
It's marked with `XFAIL`, which means, that with older version it is expected to fail. However, we report XPASS as fails, so with newer version the test passes, but we report it as a failure. As I said, we need to somehow distinguish between driver versions in LITs, to avoid this kind of confusion in future.